shared/src/api/question/schema.ts

```diff
@@ -1,41 +1,8 @@
-import * as z from "nestjs-zod/z";
-import { createZodDto } from "nestjs-zod/dto";
+import { z } from "zod";
 
-import { UserModel } from "src/zod";
-
-const passwordZodString = z
-  .string()
-  .min(8, { message: "Password must be at least 8 characters" });
-
-export const SignupSchema = UserModel.pick({
-  email: true,
-  username: true,
-}).extend({
-  password: passwordZodString,
-});
-
-export const SigninSchema = SignupSchema.pick({
-  email: true,
-  password: true,
-});
-
-export const ForgetPasswordSchema = UserModel.pick({
-  email: true,
-});
-
-const ResetPasswordSchema = SignupSchema.pick({
-  password: true,
-}).extend({ token: z.string() });
-
-export const EditableSchema = UserModel.pick({
-  email: true,
-  username: true,
-  hashRt: true,
-}).partial();
-
-export const ChangePasswordInfoSchema = z.object({
-  currentPassword: passwordZodString,
-  newPassword: passwordZodString,
+const SummaryQuerySchema = z.object({
```

Comment:
lint is giving a warning that `SummaryQuerySchema` is never used

Reply:
can ignore, will be using later on